src/domains/idpay-app/99_variables.tf

```diff
@@ -34,6 +34,15 @@ variable "domain" {
   }
 }
 
+variable "idpay_alert_enabled" {
```

Comment:
this structure is a waste, let's just define it as a simple boolean, having default value false
MoreOver, inside uat and prod (see env/* folders) you have to set this variable to true